tests/phpunit/tests/test-razorpay-route-actions.php

```diff
@@ -16,6 +17,171 @@ public function setup(): void
         $this->instance = Mockery::mock('RZP_Route_Action')->makePartial()->shouldAllowMockingProtectedMethods();
     }
 
+    public function testReverseTransfer()
+    {
+
+        $this->instance->shouldReceive('fetchRazorpayApiInstance')->andReturnUsing(
+            function () {
+                return new MockApi('key_id_2', 'key_secret2');
+        });
+
+        $_POST['transfer_id']       = 'test';
+        $_POST['reversal_amount']   = 12;
+
+        $pageUrl = admin_url('admin.php?page=razorpayTransfers&id=' . 'test');
+        $this->instance->shouldReceive('redirect')->with($pageUrl);
+
+        $response = $this->instance->reverseTransfer();
+
+        $this->assertNull($response);
+    }
+
+    public function testDirectTransfer()
```

Comment:
@garimaprachi add negative / exception block test cases

Reply:
A wp_die statement is there so the function will die because of which phpunit wont send in result
Hence it is not possible.